src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -120,6 +122,7 @@ def init_from_file(self) -> None:
             profile_jsonc = commentjson.load(fileobj)
 
         self.profiles = profile_jsonc.get("profiles", {})
+        self.schema_property = profile_jsonc.get("$schema", None)
```

Comment:
I have added the `schema_property` in `ConfigFile` class, so that it will automatically search for the `$schema` property